### PR TITLE
done. Added wc

### PR DESCRIPTION
**Pull request by mikaiyl**

### CODE REVIEW

**Comment by nlhowell12** [comment]
Think there was a slight misunderstanding.  The object of the exercise was to print the number times each word appears in the document.
E.g. 
"the": 1605,
"and": 766,
etc...

I'm also getting a "too many values to unpack" in your print_top function.

Take another look at the instructions and resubmit.   Be sure to have meaningful commit messages also.

File: wordcount.py [hunk 1]
```diff
@@ -69,12 +69,14 @@ def get_word_count( words ):
 
 def print_words( file ):
     [ words, lnum, cnum ] = read_file( file )
-    print get_word_count( words )
-    return
+    lower_words = sorted( map( lambda w: w.lower(), words ) )
```
Comment (ptrthegr8) [inline wordcount.py]: bhjvghjgjhghj

File: wordcount.py [hunk 2]
```diff
@@ -69,12 +69,14 @@ def get_word_count( words ):
...
+    for word in words:
+        print word + ": x" + str( lower_words.count( word.lower() ) )
+    return words
 
 def print_top( filename ):
     # get the words
-    [ words, lnum ] = read_file( filename )
+    [ words, lnum, cnum ] = read_file( filename )
```
Comment (ptrthegr8) [inline wordcount.py:79]: Keep in mind that you can unpack variables without enclosing them in square brackets. e.g. 
words, lnum, cnum = read_file(file)

**Comment by ptrthegr8** [comment]
Outputs expected results. PR closed. 5 points awarded.

**Pull request merged**